vyper/context/types/indexable/mapping.py

```diff
@@ -14,6 +14,9 @@
 class MappingDefinition(IndexableTypeDefinition):
     _id = "HashMap"
 
+    def __repr__(self):
+        return f"HashMap[{self.key_type}, {self.value_type}]"
```

Comment:
Just in case you ever subclass this further for other mapping types :smile: 
```suggestion
        return f"{self._id}[{self.key_type}, {self.value_type}]"
```